awx/ui/src/components/LaunchButton/LaunchButton.test.js

```diff
@@ -139,8 +139,8 @@ describe('LaunchButton', () => {
           type: 'workflow_job_template',
         }}
       >
-        {({ handleLaunch, isLaunching }) => (
-          <button type="submit" onClick={handleLaunch} disabled={isLaunching} />
+        {({ handleLaunch }) => (
+          <button type="submit" onClick={handleLaunch} disabled={true} />
```

Comment:
boolean props can just be set as flags (`<button type="submit" onClick={handleLaunch} disabled />`)